src/Microsoft.ML.CodeGenerator/Templates/Console/ModelProject.tt

```diff
@@ -3,21 +3,31 @@
 <#@ import namespace="System.Linq" #>
 <#@ import namespace="System.Text" #>
 <#@ import namespace="System.Collections.Generic" #>
+<#@ include file="PackageVersion.ttinclude" #>
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+<# if (IncludeLightGBMPackage || IncludeMklComponentsPackage || IncludeFastTreePackage){ #>
+    <StablePackageVersion><#StablePackageVersion();#></StablePackageVersion>
```

Comment:
I don't think we should add these MSBuild properties to the project. It is a little more complicated for the user.  Plus the name `StablePackageVersion` is too generic for a typical user's project. They will reference a lot of things, so having a property named "StablePackageVersion" which is only talking about ML.NET doesn't make sense.

Just put the package version directly in the `<PackageReference` element.